tests/unit/hook_packages/terraform/hooks/prepare/prepare_base.py

```diff
@@ -941,6 +943,44 @@ def setUp(self) -> None:
             "Metadata": {"SamResourceId": f"aws_apigatewayv2_integration.{self.apigwv2_integration_name}"},
         }
 
+        self.tf_apigwv2_authorizer_common_attributes: dict = {
```

Comment:
We shoudl add this authorizer to the `self.tf_json_with_root_module_only` list below for the transformation to be tested.

Reply:
Good catch